sdk/cosmos/azure-cosmos-encryption/src/test/java/com/azure/cosmos/encryption/EncryptionAsyncApiCrudTest.java

```diff
@@ -78,6 +78,21 @@ public void afterClass() {
         this.client.close();
     }
 
+    @Test(groups = {"encryption"}, timeOut = TIMEOUT)
+    public void createItem() {
```

Comment:
lets cover the newly added api in this single test , and you can change the name of test some like crudOnDifferentOverload

Reply:
Added a new test case, testing all the overloaded crud operations